### DISCUSSION TITLE
Consolidate scores button

### DISCUSSION THREAD

**Discussion by Gamingzone123** [ppy/osu#16539] [Ideas] [answered]
low priority but i thought i would put it out there; the idea is a button that would probably be in maintenance settings along with import from stable does something along the lines of:

- download scores from server for beatmaps currently stored on disk (maybe integrate a check into the F5 refresh?)
- make sure all scores are uploaded before deleting scores from disk relating to beatmaps that are not currently on disk 

the idea would be to make sure the local leaderboards show all of you scores rather than just the ones on the current device, relevant to when lazer is pushed and many players will likely get the mobile version to kill time. also, i dont quite know how it works but i imaging offloading irrelevant scores may improve performance and reduce disk usage slightly?

**Comment by peppy** [comment, marked as answer]
You'd use the global leaderboard for this, not local.
